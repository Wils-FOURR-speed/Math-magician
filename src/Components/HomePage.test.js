import React from 'react';
import renderer from 'react-test-renderer';
import Home from './HomePage';

describe('Home', () => {
  test('renders correctly', () => {
    const component = renderer.create(<Home />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});