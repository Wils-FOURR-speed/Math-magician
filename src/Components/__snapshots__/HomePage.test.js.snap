// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
<div
  className="home"
>
  <h1>
    Welcome to our Page
  </h1>
  <div
    className="intro"
  >
    <p>
       Math magician is a website where you could carry out you calculations👌😃 
    </p>
    <br />
    <p>
       You might want to try it out. So come on...😉😁 
    </p>
  </div>
</div>
`;
